src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsEFEMKernels.hpp

```diff
@@ -0,0 +1,412 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file SolidMechanicsEFEMKernels.hpp
+ */
+
+#ifndef GEOSX_PHYSICSSOLVERS_SOLIDMECHANICS_SOLIDMECHANICSEFEMKERNELS_HPP_
+#define GEOSX_PHYSICSSOLVERS_SOLIDMECHANICS_SOLIDMECHANICSEFEMKERNELS_HPP_
+
+#include "SolidMechanicsSmallStrainQuasiStaticKernel.hpp"
+#include "EFEMKernelsHelper.hpp"
+
+namespace geosx
+{
+
+namespace SolidMechanicsEFEMKernels
+{
+
+/**
+ * @brief Implements kernels for solving quasi-static equilibrium.
+ * @copydoc geosx::finiteElement::ImplicitKernelBase
+ * @tparam NUM_NODES_PER_ELEM The number of nodes per element for the
+ *                            @p SUBREGION_TYPE.
+ * @tparam UNUSED An unused parameter since we are assuming that the test and
+ *                trial space have the same number of support points.
+ *
+ * ### QuasiStatic Description
+ *
+ */
+template< typename SUBREGION_TYPE,
+          typename CONSTITUTIVE_TYPE,
+          typename FE_TYPE >
+class AssumedEnhancedStrain :
```

Comment:
This class name seems very generic, and inconsistent with the EFEM terminology used elsewhere.

Reply:
suggestions?

Reply:
perhaps `QuasiStatic`?  It looks like that would be consistent with what is done in the parent class.